api/query/query.go

```diff
@@ -145,7 +167,7 @@ func (qh queryHandler) loadSummary(testRun shared.TestRun) ([]byte, error) {
 }
 
 func getRedisKey(testRun shared.TestRun) string {
-	return "RESULTS_SUMMARY-" + strconv.FormatInt(testRun.ID, 10)
+	return "RESULTS_SUMMARY-v2-" + strconv.FormatInt(testRun.ID, 10)
```

Comment:
Just a thought - what if we make the `_v2` consistent here, since `-summary_v2.json.gz`, e.g. `RESULTS_SUMMARY_v2-`

Reply:
Seems like a good idea - changed